addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -796,6 +800,19 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         field.formatInfo.labelPosition = value;
         await this._replaceField(field);
     },
+    /**
+     * Select the textarea default value
+     */
+    selectTextareaValue: function (previewMode, value, params) {
+        this.$target[0].textContent = value;
+        this.$target[0].value = value;
+    },
+    /**
+     * Select the date as value property and convert it to the right format
+     */
+    selectValueProperty: function (previewMode, value, params) {
+        this.$target[0].value = moment.unix(value).format(params.format);
```

Comment:
I am not sure to understand: on page load you load the default value using the page format but it is already saved with a specific format? Why not saved as a timestamp?

Reply:
It is saved as a timestamp on the value attribute. Here it is the value property that needs to be changed and parsed to the format so the user see a beautiful date !